### DISCUSSION TITLE
Unrecognized function or variable 'conversionstacks_SI_SM'.

### DISCUSSION THREAD

**Discussion by AllendePrieto** [alexisInSAR/EZ-InSAR#63] [Help]
Hi Alexis!
When I try to process "interferogram stack", the program returns this message:
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/131383377/8942d66d-1022-4ba4-9c1f-627c86b2f515)

and I don't know how to continue...

**Comment by alexisInSAR** [comment]
I've sent you an e-mail
